opentelemetry-otlp/README.md

```diff
@@ -83,25 +99,31 @@ Example showing how to override all configuration options. See the
 ```rust
 use opentelemetry::{KeyValue, Tracer};
 use opentelemetry::sdk::{trace, IdGenerator, Resource, Sampler};
-use opentelemetry_otlp::{Compression, Credentials, Protocol};
+use opentelemetry_otlp::{Protocol};
 use std::time::Duration;
+use tonic::{
+    metadata::*,
+    transport::{Certificate, ClientTlsConfig},
+};
 
 fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
-    let headers = vec![("X-Custom".to_string(), "Custom-Value".to_string())]
-        .into_iter()
-        .collect();
+    let cert = std::fs::read_to_string("ca.pem")?;
+
+    let mut map = MetadataMap::new();
```

Comment:
Is it possible to pre allocate the space here with `with_capacity`? Looks like the size is fixed.

Reply:
You are right, this makes sense in this context

https://github.com/hyperium/http/blob/master/src/header/map.rs#L471